src/pages/docs/live-editor/intro-testsigma-terminal.md

```
@@ -50,7 +50,7 @@ Testsigma Terminal provides complete control over test case execution and debugg
 
    - **Installing Testsigma Terminal**
    
-   - **Editing a Test Case Using Live Editor**
+   - **Editing a Test Case Using Copilot**
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 3

(MD007, ul-indent)